### DISCUSSION TITLE
Welcome to fun-adventure-in-turkey Discussions!

### DISCUSSION THREAD

**Discussion by quillcraftsman** [PaxlavaGames/fun-adventure-in-turkey#1] [Announcements]
## 👋 Welcome!
  We’re using Discussions as a place to connect with other members of our community. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.
  * Welcome others and are open-minded. Remember that this is a community we
  build together 💪.

  To get started, comment below with an introduction of yourself and tell us about what you do with this community.

**Comment by OrPolushka** [comment]
When will the project start?

**Reply by quillcraftsman** [reply]
The project have already started, but just in the beginning.
